doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -3,7 +3,7 @@
 .. _quickstart-ubuntu:
 
 =======================================================
-Development environment quickstart guide (Ubuntu 16.04)
+Development environment quickstart guide (Ubuntu)
```

Comment:
Can someone try this as-is on Mac? Now that `conda` compilers seem to be working, there may not be a need for the Homebrew compilers, and maybe we can unify the two guides.

Reply:
According to [this](https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html), the compilers for Mac are `clang_osx-64`, `clangxx_osx-64`, and `gfortran_osx-64`. I modified the `conda install` command accordingly, but build failed.
```
ld: could not parse object file build/temp.macosx-10.9-x86_64-3.8/scipy/integrate/_quadpackmodule.o: 'Unknown attribute kind (61) (Producer: 'LLVM10.0.0' Reader: 'LLVM APPLE_1_1000.10.44.4_0')', using libLTO version 'LLVM version 10.0.0, (clang-1000.10.44.4)' file 'build/temp.macosx-10.9-x86_64-3.8/scipy/integrate/_quadpackmodule.o' for architecture x86_64
collect2: error: ld returned 1 exit status
error: Command "/Users/matthaberland/anaconda3/envs/scipydev/bin/x86_64-apple-darwin13.4.0-gfortran -Wall -g -Wall -g -undefined dynamic_lookup -bundle -Wl,-pie -Wl,-headerpad_max_install_names -Wl,-dead_strip_dylibs -Wl,-rpath,/Users/matthaberland/anaconda3/envs/scipydev/lib -L/Users/matthaberland/anaconda3/envs/scipydev/lib build/temp.macosx-10.9-x86_64-3.8/scipy/integrate/_quadpackmodule.o -L/Users/matthaberland/anaconda3/envs/scipydev/lib -L/Users/matthaberland/anaconda3/envs/scipydev/bin/../lib/gcc/x86_64-apple-darwin11.4.2/4.8.5 -L/Users/matthaberland/anaconda3/envs/scipydev/bin/../lib/gcc/x86_64-apple-darwin11.4.2/4.8.5/../../.. -L/Users/matthaberland/anaconda3/envs/scipydev/bin/../lib/gcc/x86_64-apple-darwin11.4.2/4.8.5/../../.. -Lbuild/temp.macosx-10.9-x86_64-3.8 -lquadpack -lmach -lmkl_rt -lpthread -lgfortran -o scipy/integrate/_quadpack.cpython-38-darwin.so" failed with exit status 1
```

Reply:
I think on macOS combining those compilers doesn't work without `conda-build`.

Reply:
After installing `conda-build`, the problem persisted.